apps/labs/posts/sangho-internship-blogpost.md

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Sangho's Internship at Quansight'
```

Comment:
```suggestion
title: "Sangho's Internship at Quansight"
```

Reply:
Since you are using a single quote in your title, I suggest changing to double quotes to wrap the title. YAML is good with both quotes like python and I suspect this is why you get the error on the next line.

Reply:
Can I add my mentor Victor as a reviewr?